content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -100,32 +84,63 @@ Each HTTP rule contains the following information:
 * A backend is a combination of Service and port names as described in the
-  [Service doc](/docs/concepts/services-networking/service/). HTTP (and HTTPS) requests to the
+  [Service doc](/docs/concepts/services-networking/service/) or a [custom resource backend](#resource-backends) by way of a CRD. HTTP (and HTTPS) requests to the
   Ingress that matches the host and path of the rule are sent to the listed backend.
 
-A default backend is often configured in an Ingress controller to service any requests that do not
+A `defaultBackend` is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### DefaultBackend {#default-backend}
 
-An Ingress with no rules sends all traffic to a single default backend. The default
-backend is typically a configuration option of the [Ingress controller](/docs/concepts/services-networking/ingress-controllers) and is not specified in your Ingress resources.
+An Ingress with no rules sends all traffic to a single default backend. The `defaultBackend` is conventionally a configuration option
+of the [Ingress controller](/docs/concepts/services-networking/ingress-controllers) and is not specified in your Ingress resources.
 
 If none of the hosts or paths match the HTTP request in the Ingress objects, the traffic is
 routed to your default backend.
 
+### Resource Backends {#resource-backend}
+
+A _`Resource`_ backend is an ObjectRef to another Kubernetes resource within the
+same namespace of the Ingress object. A _`Resource`_ is a mutually exclusive
+setting with Service, and will fail validation if both are specified. A common
+usage for a _`Resource`_ backend is to ingress data to a object storage backend
+with static assets.
+
+{{< codenew file="service/networking/ingress-resource-backend.yaml" >}}
+
+After creating the Ingress above, you can view it with the following command:
+
+```
+kubectl describe ingress ingress-resource-backend
+```
+
+```
+Name:             ingress-resource-backend
+Namespace:        default
+Address:
+Default backend:  APIGroup: k8s.example.com, Kind: StorageBucket, Name: static-assets
+Rules:
+  Host        Path  Backends
+  ----        ----  --------
+  *
+              /icons   APIGroup: k8s.example.com, Kind: StorageBucket, Name: icon-assets
+Annotations:  <none>
+Events:       <none>
+```
+
 ### Path Types
 
-Each path in an Ingress has a corresponding path type. There are three supported
-path types:
+Each path in an Ingress is required to have a corresponding path type. Paths
+that do not include an explicit `pathType` will fail validation. There are three
+supported path types:
 
-* _`ImplementationSpecific`_ (default): With this path type, matching is up to
-  the IngressClass. Implementations can treat this as a separate `pathType` or
-  treat it identically to `Prefix` or `Exact` path types.
+* _`ImplementationSpecific`_: With this path type, matching is up to the
```

Comment:
Same here ... please remove the "_" decorations.

Reply:
Updated, thanks!